drivers/age-swift/Sources/age-swift/age.swift

```diff
@@ -102,15 +127,17 @@ class Age{
     
     func connect(connectionParam:[String:Any], graph:String){
         self.connection = connectDatabase(connectionParam: connectionParam)
-        setUpAge(connection:self.connection!)
+        setUpAge(connection:self.connection!, graphName: graph);
     }
     
     func execSQL(statement:String){
         if self.connection == nil{
             return print("The connection is not yet established")
         }
-        return querySQL(statement: statement, connection: self.connection!)
-    }
-    
-    
-}
+        let rows = fetchSQL(statement: statement, connection: self.connection!)
+        
+        for column in rows{
+            print(column)
+        }
+    } 
+}
```

Comment:
Hi, your PR looks good, but can you correct this no newline at end of file error? Thank you

Reply:
Added a new line